Signal/src/textsecure/Messages/TSInteraction.m

```diff
@@ -81,15 +78,18 @@ - (NSString*)description{
 }
 
 - (void)saveWithTransaction:(YapDatabaseReadWriteTransaction *)transaction{
+    if (!self.uniqueId) {
+        self.uniqueId = [TSStorageManager latestTimestampWithTransaction:transaction];
```

Comment:
Yeah, I'll rename the method to something like `[TSStorageManager getAndIncrementMessageIdWithTransaction:]